lib/services/auth-service.js

```diff
@@ -0,0 +1,149 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+var jwt = require('jsonwebtoken');
+var crypto = require('crypto');
+
+module.exports = authServiceFactory;
+di.annotate(authServiceFactory, new di.Provide('Auth.Services'));
+di.annotate(authServiceFactory,
+    new di.Inject(
+        'Assert',
+        'Services.Configuration'
+    )
+);
+
+function authServiceFactory(
```

Comment:
@cgx027 @anhou As we discussed, I think we can think about this some more and make a decision later.  I'm fine with the change as is for now.